### ISSUE TITLE
Control Port Stats on decoder side issue

### ISSUE THREAD

**Issue by TheSashmo** [CESNET/UltraGrid#366]
Seems like the control ports stats when enabled only report back two channels instead of 16 when the encoded signal has 16 coming into the decoder.  Using release version, I have not tried continuous.

**Comment by mpiatka** [comment]
I've changed that a while ago in the continuous version which can now report more audio channels. Right now it's limited to 8 though. Would raising the limit to 16 be enough for you?

**Comment by TheSashmo** [comment]
Yes that would be correct.  Ideally should match what the encoder supports.

**Comment by mpiatka** [comment]
Alright, I made the default 16 and also made it configurable through `--param control-report-audio-ch-count=<count>`.
